src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
+		display: inline-block;
+	}
+
+	@LoadingMessage-padding: @size-standard;
```

Comment:
i think we should talk more about whether or not to scope component-specific less variables...i'm not sure why we don't scope them either